core/tests/dds_core_tests/src/dds_env_test.cpp

```diff
@@ -42,11 +42,11 @@ TEST(DdsEnvironmentTest, DdsEnvironmentTest) {
     dds::sub::DataReader<kpsr_dds_core::DDSEnvironmentData> datareader2(sub, topic);
 
     kpsr::YamlEnvironment yamlEnvSub;
-    yamlEnvSub.reload("greeting: \"hello\"\ntest: 123\niteration: -1\nkpsr_dds_env_key: \"test\"");
+    yamlEnvSub.updateConfiguration("greeting: \"hello\"\ntest: 123\niteration: -1\nkpsr_dds_env_key: \"test\"");
```

Comment:
We need much more testing please. In all middlewares and testing the new root node, loading multiple files. etc.

Reply:
Added additional tests for dds, zmq and yaml environments.